airflow/README.md

```diff
@@ -108,11 +108,11 @@ Connect Airflow to DogStatsD (included in the Datadog Agent) by using the Airflo
            name: "airflow.pool.open_slots"
            tags:
              pool_name: "$1"
```

Comment:
Thanks! Can you also update the tests here: https://github.com/DataDog/integrations-core/blob/master/airflow/tests/k8s_sample/datadog_values.yaml#L922 https://github.com/DataDog/integrations-core/blob/master/airflow/tests/test_check_metrics_up_to_date.py

Reply:
updated, hope i got the last one right though